src/tokens/transformed.tokens.json

```diff
@@ -21,6 +21,48 @@
       }
     }
   },
+  "background": {
+    "default-bkg": {
```

Comment:
How do you feel about changing this from "default-bkg" to "default" to match the other tokens?

Reply:
Yeeees, that would be better.